python/color_board_game.py

```diff
@@ -180,12 +175,23 @@ def tick(self):
         # elif data_buffer == bytearray("GAME_STARTING", "ASCII"):
         
     
-def main(num_of_players=1):
+def main():
+    try:
+        with open('config.yml', 'r') as file:
+            config = safe_load(file)
+    except IOError:
+        print("No config.yml file found!")
+        return
+
     boards = []
-    id = user_id
-    for x in range(0,num_of_players):
+    id = randint(100, 1234567)
```

Comment:
The user_id isn't actually supposed to be random. The random range is only there just in case more than one user forgets to change the user_id

The server will reject incoming connection requests if they share the same user_id. However, if a client is disconnected for some reason, they can use the same user_id in order to reconnect without creating a new player. It's still a bit WIP though, reconnecting works fine in the game server, but the game board handler doesn't have any logic to handle reconnecting players.